pkg/installer/src/utils.rs

```diff
@@ -88,15 +88,32 @@ pub fn save_config_value(c: &mut Cursive, k: &str, v: &str, m: bool) -> crate::e
     Ok(())
 }
 
-pub fn edit_config_value(c: &mut Cursive, k: &str, v: &str) -> crate::error::Result<()> {
+pub fn add_config_value(c: &mut Cursive, k: &str, v: &str) -> crate::error::Result<()> {
     let mut s: GlobalState = c.take_user_data().unwrap();
-    let mut vedit = s.data.map.get(k).unwrap().clone();
-    if vedit == "_____" {
-        vedit = v.to_string();
+    let mut vadd = s.data.map.get(k).unwrap().clone();
+    if vadd == "_____" {
+        vadd = v.to_string();
     } else {
-        vedit = vedit + "," + v;
+        vadd = vadd + "," + v;
     }
-    s.data.map.insert(k.to_string(), vedit);
+    s.data.map.insert(k.to_string(), vadd);
+    c.set_user_data(s);
+    Ok(())
+}
+
+pub fn remove_config_value(c: &mut Cursive, k: &str, v: &str) -> crate::error::Result<()> {
+    let mut s: GlobalState = c.take_user_data().unwrap();
+    let mut vremove: String = s.data.map.get(k).unwrap().clone()
```

Comment:
the same. don't we better call `and_then` ?

Reply:
Change it to something like this for example: let value = s.data.map.get(k).and_then(|v| Some(v.clone()));